### ISSUE TITLE
No longer works after latest patch

### ISSUE THREAD

**Issue by tonyoffermans** [skuIIs/bf4-seeder#1]
Have been using it but DICE patched the flooder issues. Many servers still use seeders though so maybe the code needs some minor modifications.

**Comment by Markus2002** [comment]
Yeah, it doesn't work on me anymore either. It always gets stuck with the first account. When I try to add a working account, the authentication always failed.

**Comment by tannobit** [comment]
@Markus2002 @tonyoffermans You guys find anything else that's working now, or any workarounds?
